WordPress/src/main/res/values/strings.xml

```diff
@@ -970,6 +970,9 @@
     <string name="aztec_editor_enabled" translatable="false">Aztec Editor enabled</string>
     <string name="aztec_editor_disabled" translatable="false">Aztec Editor disabled</string>
     <string name="aztec_editor_available" translatable="false">Aztec Editor available</string>
+    <string name="new_editor_promo_button">@android:string/ok</string>
```

Comment:
I think we should just use the `android.R.string.ok` instead of adding a resource for it as we do in many places which can be seen in this PR #5651.

If we want to add a resource for it, we should at least mark it as `translatable="false`.